VRDR/DeathRecord.cs

```diff
@@ -2956,6 +2956,92 @@ public string DateOfBirth
             }
         }
 
+        /// <summary>Decedent's Date of Birth Date Part Absent Extension.</summary>
+        /// <value>the decedent's date of birth date part absent reason</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleDeathRecord.DateOfBirthDatePartReason = "1940-02-19";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Decedent Date of Birth Date Part Reason: {ExampleDeathRecord.DateOfBirthDatePartAbsent}");</para>
+        /// </example>
+        [Property("Date Of Birth Date Part Absent", Property.Types.TupleArr, "Decedent Demographics", "Decedent's Date of Birth Date Part.", true, "http://build.fhir.org/ig/HL7/vrdr/StructureDefinition-VRDR-Decedent.html", true, 14)]
+        [FHIRPath("Bundle.entry.resource.where($this is Patient)._birthDate.extension.where(url='http://hl7.org/fhir/us/vrdr/StructureDefinition/VRDR-Partial-date-part-absent-reason')", "_birthDate")]
+        public Tuple<string,string>[] DateOfBirthDatePartAbsent
+        {
+            get
```

Comment:
Doesn't the date-part-absent extension get used in multiple places?
Could the code be factored so that the handling is in a method that is invoked by each place where the extension can be used?

Reply:
yes, I'll put it in a helper function for reuse